components/Tabs/Tabs.js

```diff
@@ -1,51 +1,62 @@
+// class Tabs {
+//   constructor(tabbies) {
+
+//   }
+// }
+
 
 class TabLink {
   constructor(element) {
     // Assign this.element to the passed in DOM element
-    // this.element;
+    this.element = element;
     
     // Get the custom data attribute on the Link
-    // this.data;
+    this.data = this.element.dataset.tab;
     
     // Using the custom data attribute get the associated Item element
-    // this.itemElement;
+    this.itemElement = document.querySelector(`.tabs-item[data-tab='${this.data}']`);
     
     // Using the Item element, create a new instance of the TabItem class
-    // this.tabItem;
-    
-    // Add a click event listener on this instance, calling the select method on click
+    this.tabItem = new TabItem(this.itemElement);
 
+    // Add a click event listener on this instance, calling the select method on click
+    this.element.addEventListener('click', this.select.bind(this));
```

Comment:
remember we can avoid the bind keyword with an arrow function, just another way of doing this
```suggestion
    this.element.addEventListener('click', () => this.select());
```